sailthru_import/sailthru_import.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import logging
+import os
+import time
+from sailthru.sailthru_client import SailthruClient
+
+"""
+Uploads a specified CSV file containing users into SailThru via the Job API endpoint.
+
+Arguments:
+    `sailthru_key`: API key
+    `sailthru_secret`: API secret
+    `csv_file_name`: CSV file name to import.
+    `list_name`: List name to import users into.
+    `report_email`: Email address which will receive report on import completion.
+
+Example usage:
+    $ sailthru_import AJKHSDKJHSADKJSH-1 KSJLAFUHDSUIH-2 input.csv "Dev Test" "bfohl@edx.org"
+
+Upload file size limit is 100 MB. csv_splitter is used to break up the export file from MailChimp.
+
+csvcut (part of csvkit) is recommended for filtering out the necessary fields:
+    csvcut -c "Email Address","activated","Age","Country","date_joined",
+    "DSTOFF","EUID","Gender","GMTOFF","id","last_login","LEID","MEMBER_RATING",
+    "REGION","TIMEZONE","Username","year_of_birth"
+    ~/Downloads/members_export_107aacb22a.csv -e ISO-8859-1 > Downloads/members_export_filtered_2.csv
```

Comment:
We probably should look at making this part of the scripted process, so that we can just point this script at a mailchimp export and walk away.